tensorboard/webapp/notification_center/_data_source/notification_center_data_source_module.ts

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {NgModule} from '@angular/core';
```

Comment:
very very minor nit: to roughly follow internal convention, I've been putting a single newline between node modules vs. modules from our source tree (relative). As formatter does not know about this, it LGTM but think about it and do what you prefer.

Reply:
oh, good catch. I like to put a newline between the imports too. Thanks for noticing this.